src/components/CheckListItem/CheckListItemDetails.tsx

```diff
@@ -22,18 +22,28 @@ const getStyles = (theme: GrafanaTheme) => ({
 interface Props {
   frequency: number;
   activeSeries?: number;
+  probeLocations?: number;
   className?: string;
   labelCount?: number;
   labels?: Label[];
   onLabelClick?: (label: Label) => void;
 }
 
-export const CheckListItemDetails = ({ frequency, activeSeries, className, labels, onLabelClick }: Props) => {
+export const CheckListItemDetails = ({
+  frequency,
+  activeSeries,
+  probeLocations,
+  className,
+  labels,
+  onLabelClick,
+}: Props) => {
   const styles = useStyles(getStyles);
   const activeSeriesMessage = activeSeries !== undefined ? `${activeSeries} active series` : null;
+  const probeLocationsMessage = probeLocations !== undefined ? `${probeLocations} locations` : null;
```

Comment:
You could check to see if there is only one location here to include/not include the `s` in location. So it would read `1 location` or `2 locations`

Reply:
That's optional, though

Reply:
added, see screenshots in description